### DISCUSSION TITLE
Using slidev with qutebrowser

### DISCUSSION THREAD

**Discussion by AckslD** [qutebrowser/qutebrowser#7554] [Q&A] [answered]
Has anyone used https://sli.dev/ with qutebrowser. I just tried their example slideshow and I just see a white page. With firefox it works. Does anyone know why and if there is some way to get it working?

**Comment by The-Compiler** [comment, marked as answer]
Seems to work for me from what I can see. What example slideshow exactly? What's your `:version` info? Can you reproduce when starting with `--temp-basedir`?

**Reply by AckslD** [reply]
Thanks @The-Compiler! It does seem to indeed be working now, not sure what was the issue when I tried.
